Neos.Flow/Classes/Package/PackageManager.php

```diff
@@ -654,28 +650,6 @@ protected function preparePackageStateConfiguration($packageKey, $packagePath, $
         ];
     }
 
-    /**
-     * Get the package version of the given package
-     * Return normalized package version.
-     *
-     * @param string $composerName
-     * @return string
-     * @see https://getcomposer.org/doc/04-schema.md#version
-     */
-    public static function getPackageVersion($composerName)
-    {
-        foreach (ComposerUtility::readComposerLock() as $composerLockData) {
-            if (!isset($composerLockData['name'])) {
-                continue;
-            }
-            if ($composerLockData['name'] === $composerName) {
-                return preg_replace('/^v([0-9])/', '$1', $composerLockData['version'], 1);
-            }
-        }
-
-        return '';
-    }
-
     /**
      * Requires and registers all packages which were defined in packageStatesConfiguration
      *
```

Comment:
The `PackageManager` refers to the no longer existing `PackageInterface::getPackageKey()`  in `unregisterPackage()` and `registerPackageFromStateConfiguration()`

Reply:
Will fix.

Reply:
Turns out `unregisterPackage` was unused and fixed the other by adjusting the return value of the PackageFactory.